plugins/copyfilelocation.lua

```diff
@@ -1,9 +1,15 @@
 -- mod-version:3
 local core = require "core"
 local command = require "core.command"
-local treeview = require 'plugins.treeview'
-local treemenu = treeview.contextmenu
-local mainmenu = require 'plugins.contextmenu'
+local config = require "core.config"
+local treeview, treemenu, mainmenu
+if false ~= config.plugins.treeview then
```

Comment:
this hurts me /lh

Reply:
I don't understand "/lh" so I don't get what is hurting you.

Anyway, just a suggestion, if you see a better way to do it, go ahead :D

Reply:
oh, heh, it means lighthearted, so it doesn't hurt me in the personal sense, just a small annoyance